Week-3/3.1/index.js

```diff
@@ -0,0 +1,17 @@
+const memoize = (fn) => {
+    let cache = {}
+    return (...args) => {
+        let cacheKey = args.map(n => n.toString() + '+').join('')
+        if (cacheKey in cache) {
+            return cache[cacheKey]
+        } else {
+            let result = args.reduce((acc, curr) => fn(acc, curr), 0);
+            cache[cacheKey] = result;
+            return result;
+        }
+    }
+}
+
+const add = (a, b) => a + b
+
+const memoizeAdd = memoize(add)
```

Comment:
Something is not right. Could you please check? 

![image](https://user-images.githubusercontent.com/13508322/178093072-21db3da6-57d6-4af8-9fcd-b2d5af729075.png)